content/applications/inventory_and_mrp/manufacturing/basic_setup/mo_costs.rst

```diff
@@ -0,0 +1,204 @@
+:guilabel:`Operation` field.
+
+Next, select a :guilabel:`Work Center` and enter the expected duration of the operation in the
+:guilabel:`Default Duration` field. The hourly cost of operating the work center and the duration of
+the operation are used to calculate the operation's cost. If no duration is recorded during
+manufacturing, the value in the :guilabel:`Default Duration` field is used instead.
+
+Finally, click :guilabel:`Save & Close` to add the operation to the |BoM| and close the
+:guilabel:`Create Operations` pop-up window. Alternatively, click :guilabel:`Save & New` to add the
+operation to the |BoM|, and open a blank :guilabel:`Create Operations` pop-up window to add another
+operation.
+
+.. seealso::
+   For a full overview of |BoM| configuration, see the documentation on :doc:`bills of materials
+   <bill_configuration>`.
+
+|MO| Overview
+=============
+
+Each |MO| has an *overview* page, which lists a variety of information about the |MO|, including
+|MO| cost and real cost. To view the overview for an |MO|, navigate to :menuselection:`Manufacturing
+app --> Operations --> Manufacturing Orders`, and select an |MO|. Then, click the
+:guilabel:`Overview` smart button at the top of the |MO|.
+
+Both the |MO| cost and real cost take into account the cost and quantity of components, as well as
+the cost of completing each work order. The overview page lists a row for each of these values, with
+the sum of them listed at the bottom of the :guilabel:`MO Cost` and :guilabel:`Real Cost` columns.
+
+Before work begins on an |MO|, the :guilabel:`MO Cost` and :guilabel:`Real Cost` columns display the
+same costs. This is the *estimated* cost of completing the |MO|.
+
+However, once work commences, the values in the :guilabel:`Real Cost` column may begin to diverge
+from the ones in the :guilabel:`MO Cost` column. This happens if a different component quantity is
+used than was listed on the |MO|, or if the duration of a work order is different than expected.
+
+Once the |MO| has been completed by clicking :guilabel:`Produce All`, the values in the
+:guilabel:`MO Cost` column update to match those displayed in the :guilabel:`Real Cost` column.
+
+.. image:: mo_costs/overview.png
+   :align: center
+   :alt: The MO Overview page.
+
+Average manufacturing cost
+==========================
+
+In addition to the cost of each individual |MO| for a product, Odoo also tracks the average cost of
+manufacturing the product, taking into account the cost of every completed |MO|. To view this,
+navigate to :menuselection:`Inventory app --> Products --> Products`, and select a product.
+
+The manufacturing cost of the product is displayed per unit of measure in the :guilabel:`Cost`
+field. The value continues to update as the costs of additional |MOs| are factored into the average
+cost.
+
+To the right of the :guilabel:`Cost` field is a :guilabel:`Compute Price from BoM` button. Click
```

Comment:
This wasn't available on every product, so I'm assuming something specific has to be in place for this to appear (e.g. the product has to have a BoM, or something along those lines...?). Either way, it might be nice to add a quick mention of this, just in case the reader encounters this issue, and is curious why they can't see such a clickable button on their product form(s).

Reply:
Also, is it a "button" or "clickable link" ... it is kinda in that "gray" area... so either option (probably) works -- but worth a quick consideration during your revisions, either way...